test/e2e/k8s_util.go

```diff
@@ -333,6 +336,44 @@ func decidePingProbeResult(stdout string, probeNum int) PodConnectivityMark {
 	return Error
 }
 
+func (k *KubernetesUtils) digUsingShort(
+	podName string,
+	podNamespace string,
+	dstAddr string,
+	useTCP bool,
+	dnsServiceIP string) (string, error) {
+
+	// Get the Pod
+	pod, err := k.clientset.CoreV1().Pods(podNamespace).Get(context.TODO(), podName, metav1.GetOptions{})
+	if err != nil {
+		log.Fatalf("Error getting pod: %v", err)
+	}
+
+	digCmd := fmt.Sprintf("dig "+"@"+dnsServiceIP+" +short %s", dstAddr)
+	if useTCP {
+		digCmd += " +tcp"
+	}
+	cmd := []string{
+		"/bin/sh",
+		"-c",
+		digCmd,
+	}
+	fmt.Printf("Running: kubectl exec %s -c %s -n %s -- %s", pod.Name, pod.Spec.Containers[0].Name, pod.Namespace, strings.Join(cmd, " "))
+	stdout, stderr, err := k.RunCommandFromPod(pod.Namespace, pod.Name, pod.Spec.Containers[0].Name, cmd)
```

Comment:
please refer to my earlier comment about running `make golangci` locally before every push, to catch linter errors

Reply:
Sorry i missed on that, my bad. I ran ```make golangci``` and found some issues that i'll address.